### DISCUSSION TITLE
qMRLab processing of DESPOT1 and DESPOT2

### DISCUSSION THREAD

**Discussion by shvdgrt** [qMRLab/qMRLab#465] [Q&A] [answered]
Hi,
I have DESPOT1 images acquired at two different flip angles (4 and 15). I used the vfa_t1 option and loaded my 4D image (from 2 flip angles) into the qMRLab. My questions are very naive, please bear with me.
1. What exactly does the qMRLab do? Does it calculate the T1 values for the image I have uploaded? If so, where is it saved?
2. For ROI mask analysis, can I use atlas generated masks?
3. Based on the answer for the first one, can spatial group analysis be done using the qMRLab?
Your response is much appreciated.
Thanks in advance
Shiv

**Comment by agahkarakuzu** [comment, marked as answer]
Hi Shiv,

1) Yes, it calculates a T1 map. You can see the [documentation](https://qmrlab.readthedocs.io/en/master/vfa_t1_batch.html):

<img width="679" alt="image" src="https://user-images.githubusercontent.com/9632322/161511963-fbf0ba2a-16b2-46c3-9514-b30e436a8157.png">


If you click the question mark buttons on GUI, you can see explanations: 

<img width="337" alt="image" src="https://user-images.githubusercontent.com/9632322/161511365-c02fdbdc-4f13-4ec1-a3dc-43dd5a4f47af.png">

If you don't provide `path data`, it will be saved to the current Matlab directory. 

2) The output T1 map will be aligned with the inputs. So your mask should spatially match the T1 map and you can easily see values per region. If you mean the ROI toolbox of qMRLab GUI, it not designed for that level of analysis, but for quick inspection/sanity checks.

3) qMRLab is mainly concerned with performing qMRI related tasks. For region/subject level statistical analyses, you'll need to use additional resources.

**Reply by shvdgrt** [reply]
Hi Agah,
Thanks for the quick reply. Much appreciated!
I understand that the results are saved as a Mat file called FitResults. Does this file contain the whole brain average T1 value?
And, if it is not too much to ask, how do I proceed further with the T1 and M0 images?
Thank you.
Shiv

**Reply by agahkarakuzu** [reply]
`FitResults.T1` has T1 values for each voxel (hence the name T1 map). If you load the file, you'll see that it has the same volumetric dimensions with your 4D input.

 Have you set your scan parameters in the protocol panel before fitting your data? 

>  how do I proceed further with the T1 and M0 images?

Depends on why you acquired this dataset, I can't really answer this question. If you have a mask generated from your DESPOT1 inputs, you can calculate simple stats per region, for example.

**Reply by shvdgrt** [reply]
Dear Agha,
Thanks for the reply. This is helpful. I will get back to you if I have any further doubts.
Best Regards
Shiv
